Tasks/powerShell/task.json

```diff
@@ -50,13 +64,25 @@
             "required":false,
             "helpMarkDown": "Current working directory when script is run.  Defaults to the folder where the script is located.",
             "groupName":"advanced"
+        },
+        {
+            "name": "script",
+            "type": "multiLine",
+            "label": "Script",
+            "defaultValue": "Write-Host 'Hello World'",
+            "required": true,
+            "helpMarkDown": "Write your scripts and have fun",
+            "visibleRule": "type = InlineScript",
+            "properties": {
+                "rows": 5
+            }
         }
     ],
     "instanceNameFormat": "Powershell: $(scriptName)",
     "execution": {
-        "PowerShellExe": {
-            "target": "$(scriptName)",
-            "argumentFormat": "$(arguments)",
+        "PowerShell": {
```

Comment:
You should not change the handler. As I mentioned previously, in addition to the change you are going after with this PR, this task needs additional improvements/thought put into it. Switching the handler is just going to make it harder to untangle the mess the current state of the task is in. Your proposed change busts compat. It doesn't handle last exit code the same. It doesn't run the users script in the global scope. It uses the PowerShell handler which runs in-proc in the worker process. We are trying to move away from this handler to a new powershell handler that runs out of process. For the in-proc reason alone, you definitely should not make this change. Part of the mess that needs to be untangled with the current task is the issue with external tools writing to STDERR which causes the handler to create an error issue and fail the task. The direction you are going with this change makes it harder to untangle that current mess. The change you are making here gives the script access to the OAuth token. Ad hoc scripts are not supposed to have access to the OAuth token. There's probably other things that need to be carefully considered as well... the change you are making here also changes how write-verbose/debug is handled.